lib/rubygems/security/signer.rb

```diff
@@ -154,7 +170,7 @@ def sign data
   # be saved as ~/.gem/gem-public_cert.pem.expired.%Y%m%d%H%M%S where the
   # expiry time (not after) is used for the timestamp.
 
-  def re_sign_key # :nodoc:
+  def re_sign_key(expiration_length: Gem::Security::ONE_YEAR) # :nodoc:
```

Comment:
This change is using 2 definitions for the default expiry time. Could we use `Gem::Security::ONE_YEAR` instead of having a `365` constant defined?

Reply:
```Gem::Security::ONE_YEAR``` is not the same as `365` and we want to be able to control de days, does that make sense?